pkg/webhook/webhook_init.go

```diff
@@ -20,13 +20,50 @@ import (
 	"knative.dev/pkg/logging"
 )
 
-const WEBHOOK_INSTALLERSET_LABEL = "validating-defaulting-webhooks.operator.tekton.dev"
-const POD_NAMESPACE_ENV_KEY = "SYSTEM_NAMESPACE"
+const (
+	// deprecated label, used in old versions
+	// keeps this reference to remove the existing webhook installersets
+	DEPRECATED_WEBHOOK_INSTALLERSET_LABEL = "validating-defaulting-webhooks.operator.tekton.dev"
+
+	// this label is used to terminate the created webhook installerset on graceful termination
+	// use unique name, to identify the resource created by this pod
+	WEBHOOK_UNIQUE_LABEL = "operator.tekton.dev/webhook-unique-identifier"
+
+	// primary label values to track webhook installersets
+	labelCreatedByValue        = "operator-webhook-init"
+	labelInstallerSetTypeValue = "operatorValidatingDefaultingWebhook"
```

Comment:
Do we still require these labels even after adding `WEBHOOK_UNIQUE_LABEL`

Reply:
yaa lets keep them for the type, and podName for the respective pods created one